src/nft/components/explore/CarouselCard.tsx

```diff
@@ -209,12 +194,13 @@ export const CarouselCard = ({ collection, onClick }: CarouselCardProps) => {
               return null
             }
             return (
-              <MarketplaceRow
-                key={`CarouselCard-key-${collection.address}-${marketplace.marketplace}`}
-                marketplace={MARKETS_ENUM_TO_NAME[market]}
-                listings={marketplace.count.toString()}
-                floorInEth={marketplace.floorPrice.toString()}
-              />
+              <>
+                <TableElement>{MARKETS_ENUM_TO_NAME[market]}</TableElement>
+                <TableElement>
+                  {formatNumberOrString(marketplace.floorPrice, NumberType.NFTTokenFloorPrice)} ETH
+                </TableElement>
+                <TableElement>{marketplace.count.toString()}</TableElement>
```

Comment:
Shouldn't this also be run through the formatter?

Reply:
i also don't think you need a toString

Reply:
Yeah, that is definitely not needed lol